tests/integration/standard/test_control_connection.py

```diff
@@ -34,6 +34,10 @@ def setup_module():
 
 class ControlConnectionTests(unittest.TestCase):
     def setUp(self):
+        if PROTOCOL_VERSION < 3:
+            raise unittest.SkipTest(
+                "Native protocol 3,0+ is required for UDTs using %r"
+                % (PROTOCOL_VERSION,))
```

Comment:
This skips all tests in the class for protocol version < 3. Did you intend to raise from a specific test?